examples/main.cpp

```diff
@@ -0,0 +1,13 @@
+/*
+ * See LICENSE for more information about licensing
+ *
+ * Copyright 2023
+ * Author: Luis G. Leon-Vega <luis.leon@ieee.org>
+ *         Diego Arturo Avila Torres <diego.avila@uned.cr>
+ *
+ */
+#include "src/cynq.hpp"
```

Comment:
Actually, we should respect the convention. You should have "cynq/cynq.hpp". We don't want to mix the project headers with other headers.